frontend/src/main/java/olumofe/raphael/frontend/CarController.java

```diff
@@ -0,0 +1,61 @@
+package olumofe.raphael.frontend;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.*;
+import starter.project.domain.Car;
+
+import java.util.*;
+
+@RestController
+public class CarController {
+
+    private List<Car> carNewList = new ArrayList<>();
+
+    @PostMapping(path = "/car", consumes = {"application/json"})
+    public List<Car> addCar(@RequestBody Car car) {
+        carNewList.add(car);
+        return carNewList;
+    }
+    @GetMapping(path = "/car/{carId}")
+    public Car getCars(@PathVariable("carId") int carId) {
+        for (Car car : carNewList) {
+            if (car.getId() == (carId)) {
+                Car returnedCar = car;
```

Comment:
We can simplify this to one line without using a variable!
Don't always listen to intelliJ! :P